### PR TITLE
Modal component inside

### PR DESCRIPTION
**Pull request by kirti2602**
Feat: -
Modal component implemented along with functionality using JS

### CODE REVIEW

File: components/text-utilities/text-utilities.css [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+    flex-direction: column;
+}
+
+.biggest-heading{
+    font-size: 2rem;
+}
+
+.gray-text{
+    color: var(--gray);
+    font-size: larger;
```
Comment (bhavyaj12) [inline components/text-utilities/text-utilities.css:14]: I think Kanishk had mentioned not to use larger, instead you could use variables or even utility classes for text sizes

**Review by bhavyaj12 — APPROVED** [review]
Seems good, other than that it's not responsive on my phone. But that can be worked on later. Code is good to go.

**Comment by kirti2602** [comment]
> Seems good, other than that it's not responsive on my phone. But that can be worked on later. Code is good to go. 

Yes I do need to work on responsiveness after getting done with it. Thanks for your feedback

**Review by sakshi006 — APPROVED** [review]
Good to go

**Pull request merged**